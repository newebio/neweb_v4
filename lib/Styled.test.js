"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const React = require("react");
// tslint:disable-next-line:no-implicit-dependencies
const react_test_renderer_1 = require("react-test-renderer");
const Styled_1 = require("./Styled");
describe("Styled::tests", () => {
    it("component should create css from props and add it to render", () => {
        const el = react_test_renderer_1.create(React.createElement(Styled_1.default, {
            styles: {
                "button": {
                    color: "blue",
                },
                "ul li>a": {
                    color: "red",
                },
            },
        }));
        expect(el.toJSON()).toMatchSnapshot();
    });
});
