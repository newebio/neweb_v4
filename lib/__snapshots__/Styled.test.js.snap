// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled::tests component should create css from props and add it to render 1`] = `
<div
  className="s__s1"
>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<style type=\\"text/css\\">.s__s1 button {
  color: blue;
}
.s__s1 ul li>a {
  color: red;
}</style>",
      }
    }
  />
</div>
`;
